tests/0104-fetch_from_follower_mock.c

```diff
@@ -111,6 +110,11 @@ static void do_test_offset_reset(const char *auto_offset_reset) {
         else
                 test_consumer_poll(auto_offset_reset, c, 0, 1, 0, msgcnt, NULL);
 
+        /* send another batch of messages to ensure the consumer isn't stuck */
```

Comment:
I think it's a good idea to expand on why exactly we add this and why the consumer was getting stuck, alternatively, we can provide the issue # . Without that, it'll be a bit difficult for future readers.